src/Servers/IIS/IIS/test/FunctionalTest.props

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <HelixTimeout>00:45:00</HelixTimeout>
+    <HelixTimeout Condition="$(HelixTargetQueue.StartsWith('Windows.10.Amd64'))">01:00:00</HelixTimeout>
```

Comment:
Seems ok to just make this 1 hour too?

Reply:
Again, we don't want to tie up the agents more than necessary